solc/CommandLineInterface.cpp

```diff
@@ -424,13 +434,9 @@ void CommandLineInterface::handleGasEstimation(string const& _contract)
 
 void CommandLineInterface::readInputFiles()
 {
-	solAssert(!m_standardJsonInput.has_value(), "");
+	solAssert(!m_standardJsonInput.has_value());
 
-	if (
-		m_options.input.mode == InputMode::Help ||
-		m_options.input.mode == InputMode::License ||
-		m_options.input.mode == InputMode::Version
-	)
+	if (std::set<InputMode>{InputMode::Help, InputMode::License, InputMode::Version}.count(m_options.input.mode) == 1)
```

Comment:
To be honest, I think the previous version was much more readable :)

Reply:
We'll get `set::contains` when™ we upgrade to c++20 :)

Reply:
We have our [own version in `CommonData.h`](https://github.com/ethereum/solidity/blob/develop/libsolutil/CommonData.h#L441-L445).

But it would still look pretty much the same. Not a big deal, just pointing out that sometimes the stupid way to do it is cleaner :)